src/ResourceManager/Consumption/Commands.Consumption/help/AzureRM.Consumption.md

```diff
@@ -14,3 +14,35 @@ The topics in this section document the Azure PowerShell cmdlets for Azure Consu
 ### [Get-AzureRmConsumptionUsageDetail](Get-AzureRmConsumptionUsageDetail.md)
 Get usage details of the subscription.
 
+## AzureRM.Consumption Cmdlets
```

Comment:
Remove this line.

Reply:
Remove all extra lines of `AzureRM.Consumption Cmdlets`